apps/web-giddh/src/app/contact/aging-dropdown/aging.dropdown.component.ts

```diff
@@ -32,6 +32,17 @@ export class AgingDropdownComponent implements OnDestroy {
     constructor(private store: Store<AppState>, private toasty: ToasterService, private agingReportActions: AgingReportActions) {
         this.setDueRangeRequestInFlight$ = this.store.pipe(select(s => s.agingreport.setDueRangeRequestInFlight), takeUntil(this.destroyed$));
     }
+    
+    /**
+     * This will use for respond after angular checks the content projected into the directive or component.
+     *
+     * @memberof AgingDropdownComponent
+     */
+    public ngAfterContentChecked(): void {
```

Comment:
⚠️ `ngAfterContentChecked` is triggered after every change detection in Angular, try to avoid this as it is called redundantly which will cause your code to execute multiple times. Read more about [here](https://stackoverflow.com/a/44615655/3298390).